configure.ac

```diff
@@ -7072,3 +7077,4 @@ if test "x$dependency_warning" = "xyes"; then
 fi
 
 # vim: set fdm=marker sw=2 sts=2 ts=2 et :
+
```

Comment:
Do you think you could filter out this newline?

Reply:
When I get rid of this, the build fails on cygwin with:
/usr/bin/m4:configure.ac: ERROR: end of file in comment

Is it ok to leave it? As far as I can tell, adding the line doesn't affect builds on Linux.